src/utils/checkers/isSameDay.js

```diff
@@ -0,0 +1,8 @@
+import { getDateNthDaysAway } from "../getters/getDateNthDaysAway.js";
+
+export function isSameDay(firstDate, secndDate) {
+	const firstDateMonTime = getDateNthDaysAway(firstDate, 0).getTime();
+	const secndDateMonTime = getDateNthDaysAway(secndDate, 0).getTime();
```

Comment:
typo

Reply:
not clear naming: getDateNthDaysAway

Reply:
In VS Code I use "Code Spell Checker" plugin. Saves my day.